src/popover/types.js

```diff
@@ -104,7 +106,10 @@ export type BasePopoverPropsT = {
   /** If true, focus will shift back to the original element that triggered the popover
    * Be careful with elements that open the popover on focus (e.g. input) this will cause the popover to reopen on close!
    */
-  returnFocus?: boolean,
+  returnFocus?:
+    | boolean
+    | FocusOptions
```

Comment:
It's a bit confusing why `FocusOptions` can be passed to both `returnFocus`, and to a new, dedicated prop `autoFocusOptions`. It looks like you are mirroring how the API is structure in the source code for `FocusLock`, although I'm not clear on why they've done it that way as well. Is there a good reason for us to do this? If yes, can we add a comment to address the possible confusion?

Reply:
yeah, I am just mimicing the prop structure of react-focus-lock here. My reason for doing it here too is to provide the same capability to the baseweb users as the react-focus-lock provides